### DISCUSSION TITLE
Github Copilot connect to server error in vs code

### DISCUSSION THREAD

**Discussion by Imran-web** [community/community#52945] [Copilot Conversations]
### Select Topic Area

Bug

### Body

[INFO] [auth] [2023-04-16T03:14:16.767Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-16T03:14:16.773Z] GitHub Copilot could not connect to the server. Extension activation failed: "No access to GitHub Copilot found."

I'm getting this error for the past 3 days i have a subscription. Please help me to resolve this issue

**Comment by Akash1134** [comment]
Hi @Imran-web ,

Sorry that you're unable to use Copilot and getting an error 403.

A common cause of a 403 error is an attempt to authenticate to Copilot with a GitHub account that does not have a Copilot subscription. For example, if you have more than one GitHub account (one for work and a personal one), you might inadvertently be signed into the account without a Copilot subscription when trying to authenticate.

In VS Code, please use the accounts menu in the lower left-hand sidebar to log out of GitHub, and then log back in, making sure you're signing into the correct GitHub account.

If that doesn't resolve your error, please reach out to our [Support](https://support.github.com/contact) team for account-specific help.
